contracts/cheddar_5x5_tic_tac_toe/src/lib.rs

```diff
@@ -461,17 +451,27 @@ impl Contract {
             "Cannot stop. Game in progress"
         );
         self.games.remove(game_id);
+        return Some(game_to_store.game_result)
     }
 
-    pub fn claim_timeout_win(&mut self, game_id: &GameId) {
+    pub fn claim_timeout_win(&mut self, game_id: &GameId) -> Option<GameResult> {
         let game: Game = self.internal_get_game(&game_id);
         let player = env::predecessor_account_id();
         if game.claim_timeout_win(player.clone()) == false {
```

Comment:
`claim_timeout_win` should take a reference, then we don't need to do `player.clone()`, and we will do `&player`